bundles/org.openhab.core.config.discovery.mdns/src/main/java/org/openhab/core/config/discovery/mdns/internal/MDNSDiscoveryService.java

```diff
@@ -64,6 +66,11 @@
 
     private final MDNSClient mdnsClient;
 
+    /*
```

Comment:
javadoc missing character:
```suggestion
    /**
```

Reply:
Ok.